### DISCUSSION TITLE
New user quota

### DISCUSSION THREAD

**Discussion by amgschnappi** [immich-app/immich#9768] [Q&A]
How to set quotas for new users?
i wud like to set a 10gb qouta for any newly registered users.

**Comment by bo0tzz** [comment]
Setting a default quota with the built-in auth is not supported. If you're using oauth, you can find it in the oauth settings. 

![kcs1Ae](https://github.com/immich-app/immich/assets/11602424/60f21595-2fc3-4d17-9e08-9d81d5a75ae9)
